src/main/org/apache/tools/ant/util/SymbolicLinkUtils.java

```diff
@@ -30,6 +30,8 @@
  * a symbolic link based on the absent support for them in Java.
  *
  * @since Ant 1.8.0
+ * @deprecated Starting Ant 1.10.2, this utility is deprecated in favour of the symlink
+ *              support introduced in Java {@link java.nio.file.Files} APIs
```

Comment:
the class is used all over the code base (well, at least in a few different places :-). Do you intend to change those other occurrences as well?

Reply:
>> Do you intend to change those other occurrences as well?

I do intend to do necessary changes related to this, in subsequent commit(s), when I get a chance. I didn't want to create one large PR with too many unrelated changes. I added this deprecation note just so that any new code doesn't end up using this class, now that we are on Java 7. But I think, it's better to add this deprecation note when I do get to changing references to this class in some other commits. So I have undone this change and updated the PR.

Reply:
sounds good to me, thanks. I prefer small changes myself as well.